share/ext/luajit/build_win32.bat

```diff
@@ -66,7 +66,7 @@ pushd %SOURCE_TARGET%
 if "%platform%" == "x64" goto :BUILD_X64
 
 :BUILD_X32
-cmd "/C msvcbuild.bat nogc64 static dummy"
+cmd "/C msvcbuild.bat static dummy"
```

Comment:
`msvcbuild.bat` has changed in LuaJIT and `nogc64` is only used in 64-bit builds. In `msvcbuild.bat` it is explained as:

`nogc64   disable LJ_GC64 mode for x64`